src/utils.js

```diff
@@ -103,13 +52,14 @@ const storeUser = (app, user) => {
  * @param {object} User - The User scope class.
  */
 const restoreUser = (app, User) => {
-  const userData = localStorage
-   ? readStorage('scanthng-' + app.id)
-   : readCookie('scanthng-' + app.id);
+  if (!localStorage) throw new Error('Cannot restore user, localStorage is not available');
```

Comment:
There was a time when Safari / mobile safari incognito prevents the use of local storage.

https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API#private_browsing_incognito_modes